service/worker/workerdeployment/workflow.go

```diff
@@ -130,6 +131,14 @@ func (d *WorkflowRunner) run(ctx workflow.Context) error {
 		return err
 	}
 
+	if err := workflow.SetUpdateHandler(
```

Comment:
I'm not sure if this `SetUpdateHandler` call needs to be in a patch too. Like, does `SetUpdateHandler` create a wf event?